src/front/app/features/create/components/validate.js

```diff
@@ -0,0 +1,19 @@
+const validate = (values) => {
+    const errors = {};
```

Comment:
Disagree, each field should have only one validation failure, othercase which of validation failure should be shown for field?